minecraft_atomizer.py

```diff
@@ -6,27 +6,20 @@
 (e.g., 4 for torch), and the items along with the quantities of those items
 required to create those items must be placed in the same directory as this
 script. Such a file can be obtained on my github TODO: Add link.
-
```

Comment:
Please add these blank lines back to the documentation to conform with the numpy style